logstash-core/lib/logstash/api/commands/node.rb

```diff
@@ -62,8 +63,8 @@ def pipeline(pipeline_id, options={})
             metrics.merge!(extended_stats)
           end
           metrics
-        rescue
-          {}
+        rescue LogStash::Instrument::MetricStore::MetricNotFound
```

Comment:
Partially limiting the scope of failure here to the same scope that is managed by most other endpoints or endpoint-fragments. If we are getting something other than a `MetricNotFound`, we should error appropriately so we can find and fix the actual issue.